kube_apiserver_metrics/datadog_checks/kube_apiserver_metrics/kube_apiserver_metrics.py

```diff
@@ -15,6 +15,7 @@
     'go_threads': 'go_threads',
     'go_goroutines': 'go_goroutines',
     'APIServiceRegistrationController_depth': 'APIServiceRegistrationController_depth',
+    # Deprecated in 1.22 (replaced by apiserver_storage_objects)
```

Comment:
can we regenerate these metrics from the new one?
I'm wondering if we should provide a migration path.